lib/node_modules/@stdlib/ndarray/some-by/test/test.assign.js

```diff
@@ -0,0 +1,1249 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var Float64Array = require( '@stdlib/array/float64' );
+var ndarray = require( '@stdlib/ndarray/ctor' );
+var empty = require( '@stdlib/ndarray/empty' );
+var ndarray2array = require( '@stdlib/ndarray/to-array' );
+var scalar2ndarray = require( '@stdlib/ndarray/from-scalar' );
+var assign = require( './../lib/assign.js' );
+
+
+// FUNCTIONS //
+
+/**
+* Callback function.
+*
+* @private
+* @param {*} value - array element
+* @returns {boolean} test result
+*/
+function clbk1( value ) {
+	return value > 0;
+}
+
+/**
+* Callback function.
+*
+* @private
+* @param {*} value - array element
+* @returns {boolean} test result
+*/
+function clbk2( value ) {
+	this.count += 1; // eslint-disable-line no-invalid-this
+	return value > 0;
+}
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof assign, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function throws an error if provided a first argument which is not an ndarray-like object(n_scalar)', function test( t ) {
+	var values;
+	var y;
+	var i;
+
+	y = empty( [], {
+		'dtype': 'bool'
+	});
+
+	values = [
+		'5',
+		5,
+		NaN,
+		true,
+		false,
+		null,
+		void 0,
+		{},
+		[],
+		function noop() {}
+	];
+
+	for ( i = 0; i < values.length; i++ ) {
+		t.throws( badValue( values[ i ] ), TypeError, 'throws an error when provided ' + values[ i ] );
+	}
+	t.end();
+
+	function badValue( value ) {
+		return function badValue() {
+			assign( value, 1, y, clbk1 );
+		};
+	}
+});
+
+tape( 'the function throws an error if provided a first argument which is not an ndarray-like object(n_ndarray)', function test( t ) {
```

Comment:
```suggestion
tape( 'the function throws an error if provided a first argument which is not an ndarray-like object (n=ndarray)', function test( t ) {
```

Here and everywhere else. We are not consistent with this, but the `n=<type>` convention is used elsewhere.